### PR TITLE
DELIA-47718 : fixed setCollectionFrequency of FrameRate and DELIA-47719 - Checking the minimum frequency value

### PR DESCRIPTION
**Pull request by MManja792**

### CODE REVIEW

File: FrameRate/FrameRate.cpp [hunk 1]
```diff
@@ -85,15 +85,24 @@ namespace WPEFramework
...
+			if( fpsFrequencyInMilliseconds > = 100) // make sure min freq is 100 and not less than that.
+			{
+        	    		setCollectionFrequency(fpsFrequencyInMilliseconds);
+            			returnResponse(true);
+			}
+            	}
+		}
+	     catch(...)
+	     {
+		returnResponse(false);
```
Comment (anand-ky) [inline FrameRate/FrameRate.cpp:102]: Pls catch the exception and add some logging and return error indicating that the value is invalid.

File: FrameRate/FrameRate.cpp [hunk 2]
```diff
@@ -85,15 +85,24 @@ namespace WPEFramework
...
 
             LOGINFOMETHOD();
+            try{
+            	if (parameters.HasLabel("frequency"))
+            	{
+            		int fpsFrequencyInMilliseconds = DEFAULT_FPS_COLLECTION_TIME_IN_MILLISECONDS;
+	                fpsFrequencyInMilliseconds = std::stod(parameters["frequency"].String());
+			if( fpsFrequencyInMilliseconds > = 100) // make sure min freq is 100 and not less than that.
+			{
+        	    		setCollectionFrequency(fpsFrequencyInMilliseconds);
```
Comment (anand-ky) [inline FrameRate/FrameRate.cpp:95]: Add logging to indicate that the value is less than 100.

**Review by anand-ky — CHANGES_REQUESTED** [review]

**Comment by MManja792** [comment]
Added the logs on exception, invalid parameter and invalid value.

File: FrameRate/FrameRate.cpp (hunk 1 above)
Comment (MManja792) [reply FrameRate/FrameRate.cpp]: Added the logs

**Review by anand-ky — APPROVED** [review]

**Pull request merged**